templates/zerver/help/include/event-filtering-instruction.md

```diff
@@ -0,0 +1,12 @@
+(Optional) To filter the events that trigger the notifications, you can append
+either (or both) `&only_events=["an_event","another_event"]` or
+`&exclude_events=["an_event","another_event"]`
+with an arbitrary number of supported events to the URL.
```

Comment:
To make the line breaks/wrapping nicer, I would change the example event names to something shorter. And I would adjust the sentence wording a little bit:

> either `&only_events=["event_a","event_b"]` or `&exclude_events=["event_a","event_b"]` (or both, with different events) to the URL with an arbitrary number of supported events.